sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -63,6 +63,8 @@ struct sof_sdw_codec_info {
 		     struct sof_sdw_codec_info *info,
 		     bool playback);
 
+	int (*exit)(struct device *dev);
+	bool is_probe;
```

Comment:
@bardliao And this should be "is_probed". Hmm, and can we just drop this and require the exit/cleanup handlers to be robust they can handle the case if cleanup is called in the case probe is not done...?

Reply:
The purpose of is_probed flag is to prevent a callback function from a non existing codec being called. Not every codecs in the codec_info_list are used. i.e. sof_sdw_rt711_init will not be invoked if there is no rt711 on the acpi mach table. That's why I need a flag to record it.